client/plots/plot.brainImaging.js

```diff
@@ -0,0 +1,50 @@
+export default async function (termdbConfig, dslabel, queryKey, sample, holder, genomeObj, _overrides = {}) {
```

Comment:
Let's move all the brain imaging files to a subfolder:

Eg: `client/plots/brainimaging/plot.brainimaging.js`

All the other files should be moved to this subfolder.

Reply:
ok to do it later

Reply:
plot.***.js has to be in plots folder now (they are imported by plot.app.js), can change it later